### ISSUE TITLE
[Accessibility] A11y_.Net Editor: The shortcuts for the actions "Add Cursor Above" and "Add Cursor Below" i.e."Ctrl+Alt+UpArrow" and "Ctrl+Alt+DownArrow" are not performing the appropriate action when used.

### ISSUE THREAD

**Issue by ChetanDumbala** [dotnet/try#122]
**Environment Details**
https://docs.microsoft.com/en-us/dotnet/csharp/quick-starts/hello-world?tutorial-step=4User
Browser: Edge (41.16299.402.0)
Chrome (67.0.3396.87)

**Additional Details:**
MAS Violated: MAS2.1.1

**Repro steps:**
Hit the URL https://docs.microsoft.com/en-us/dotnet/csharp/quick-starts/hello-world?tutorial-step=4User
Tab till “Enter focus mode” and press enter
Write a Sample program
Now try the keyboard shortcuts for the actions "Add Cursor Above" and "Add Cursor Below" i.e."Ctrl+Alt+UpArrow" and "Ctrl+Alt+DownArrow" and verify whether the corresponding action is getting performed or not.

**Actual Result:**
The Shortcuts "Ctrl+Alt+UpArrow" and "Ctrl+Alt+DownArrow" are not performing the intended action instead the screen orientation is getting changed because of the shortcut

**Expected Result:**
All the shortcuts should work properly.

**User Impact:**
A disabled user wouldn't be able to avail the functionality as a particular shortcut is not working.

**MAS Reference:**
https://microsoft.sharepoint.com/teams/msenable/_layouts/15/WopiFrame.aspx?sourcedoc={8492c4eb-c179-40ae-8777-cd044ed725a2}

[The shortcuts for the actions Add Cursor Above and Add Cursor Below i.e.Ctrl+Alt+UpArrow and Ctrl+Alt+DownArrow are not performing the appropriate action when used..pptx](https://github.com/dotnet/try/files/2191951/The.shortcuts.for.the.actions.Add.Cursor.Above.and.Add.Cursor.Below.i.e.Ctrl%2BAlt%2BUpArrow.and.Ctrl%2BAlt%2BDownArrow.are.not.performing.the.appropriate.action.when.used.pptx)

**Comment by jonsequitur** [comment]
@ChetanDumbala Please reopen this issue under https://github.com/Microsoft/monaco-editor.

**Comment by LadyNaggaga** [comment]
1.	Importance of experience (High or Low Impact). 
Impact: Low we haven't received any customer complaints directly to the service. The .NET online editor is the service behind editor and we do not own or control the UI experience. 
Describe scenario: Please the description above
2.	Quality of Experience (High or Low Impact). 
Impact: Low we haven't received any customer complaints directly to the service. The .NET online editor is the service behind editor and we do not own or control the UI experience. 
3.	[Link to AAT ](https://cela-ra-aat-prod.azurewebsites.net/#/workflow/93711e75-9cb7-4061-b1e9-53177a087cae)
4.	Any alternate solutions or technologies evaluated
Not applicable 
5.	Ensure all evidence for hand-off to external team is provided in the bug
Issue has been opened on the Monaco repo 
https://github.com/Microsoft/monaco-editor/issues/1181

**Comment by umasarath52** [comment]
This was discussed with Ryan and approved to close as this in non blocking scenario.

**Comment by soumyatippapuram** [comment]
#A11yMAS

**Comment by soumyatippapuram** [comment]
GitHubTags: #A11yTCS; #A11yMAS; #C+E-H; #DesktopWeb; #Win10; #A11y_.NET Online REPL View; #MAS2.1.1; #Keyboard; #.NET Online REPL; #Edge; #Chrome
